lib/vue-simple-suggest.vue

```diff
@@ -377,6 +376,8 @@ export default {
     },
     async showSuggestions () {
       if (this.suggestions.length === 0 && this.minLength <= this.textLength) {
+        // try show misc slots while researching
+        this.showList()
```

Comment:
This makes sense only if `this.miscSlotsAreEmpty()` is `false`.

Reply:
This line is the replacement for [L573](https://github.com/KazanExpress/vue-simple-suggest/pull/186/files#diff-2f0ba4d49aa01cd5ffcfa361ee366128L573)
I think it should do the same but is located in a more suitable place, in my opinion.
Do you think something is wrong with it?

Reply:
Not wrong per se. It's just that L573 was placed in `!this.miscSlotsAreEmpty()` condition, while this one is not.

Reply:
But `showList()` already has this condition inside :)